pkg/destroy/openstack/openstack_deprovision.go

```diff
@@ -541,6 +544,73 @@ func deleteTrunks(opts *clientconfig.ClientOpts, filter Filter, logger logrus.Fi
 	return len(allTrunks) == 0, nil
 }
 
+func deleteLoadBalancers(opts *clientconfig.ClientOpts, filter Filter, logger logrus.FieldLogger) (bool, error) {
+	logger.Debug("Deleting openstack load balancers")
+	defer logger.Debugf("Exiting deleting openstack load balancers")
+
+	conn, err := clientconfig.NewServiceClient("load-balancer", opts)
+	if err != nil {
+		logger.Fatalf("%v", err)
+		os.Exit(1)
+	}
+
+	newallPages, err := apiversions.List(conn).AllPages()
+	if err != nil {
+		logger.Debug("Unable to list api versions: %v", err)
+	}
+
+	allAPIVersions, err := apiversions.ExtractAPIVersions(newallPages)
+	if err != nil {
+		logger.Debug("Unable to extract api versions: %v", err)
+	}
+
+	var octaviaTagSupport bool
+	octaviaTagSupport = false
+	for _, apiVersion := range allAPIVersions {
+		if apiVersion.ID >= "v2.5" {
+			octaviaTagSupport = true
+		}
+	}
+
+	tags := filterTags(filter)
+	var listOpts loadbalancers.ListOpts
+	if octaviaTagSupport {
```

Comment:
what would happen if the OpenStack cloud was upgraded after the installation of OCP? Could it be that octavia was not having support for tags, and then after the upgrade it has, but the resources are not tagged but using description? Perhaps better to run the else part regardless of the octavia version?

Reply:
Good point. I've updated the PR with you suggestion. Thanks!